Sources/SwiftBSON/ExtendedJSONEncoder.swift

```diff
@@ -29,14 +40,14 @@ public class ExtendedJSONEncoder {
         // T --> BSON --> JSONValue --> Data
         // Takes in any encodable type `T`, converts it to an instance of the `BSON` enum via the `BSONDecoder`.
         // The `BSON` is converted to an instance of the `JSON` enum via the `toRelaxedExtendedJSON`
-        // or `toCanonicalExtendedJSON` methods on `BSONValue`s (depending on the `mode`).
+        // or `toCanonicalExtendedJSON` methods on `BSONValue`s (depending on the `format`).
         // The `JSON` is then passed through a `JSONEncoder` and outputted as `Data`.
         let encoder = BSONEncoder()
         encoder.userInfo = self.userInfo
         let bson: BSON = try encoder.encodeFragment(value)
 
         let json: JSON
-        switch self.mode {
+        switch self.format._format {
         case .canonical:
             json = bson.bsonValue.toCanonicalExtendedJSON()
         case .relaxed:
```

Comment:
the docstrings for `encode` and `encodeBuffer` have refereneces to `self.mode` that need to be updated.

Reply:
good catch, fixed 👍